### PR TITLE
docs: translate to Spanish intro.md of package-structure-code

### PR DESCRIPTION
**Pull request by RobPasMue**
Related to #313

### CODE REVIEW

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 1]
```diff
@@ -1487,21 +1490,28 @@ msgid ""
...
-msgstr ""
+msgstr "✨ 1. Estructura de archivos en el paquete ✨"
 
 #: ../../package-structure-code/intro.md:23
 msgid ""
 "src layout, flat layout and where should tests folders live? No matter "
 "what your level of packaging knowledge is, this page will help you decide"
 " upon a package structure that follows modern python best practices."
 msgstr ""
+"¿Distribución del código fuente, distribución plana y dónde deben vivir las carpetas de tests? "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1508]:
```suggestion
"¿Distribución con estructura 'src', distribución plana y dónde deben vivir las carpetas de tests? "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 2]
```diff
@@ -1510,72 +1520,90 @@ msgid ""
...
+msgstr "✨ 4. Agregar metadatos ✨"
 
 #: ../../package-structure-code/intro.md:45
 msgid ""
 "Learn how to add project metadata to your Python package to support both "
 "filtering on PyPI and also the metadata that a package installer needs to"
 " build and install your package."
 msgstr ""
+"Aprenda cómo agregar metadatos de proyecto a su paquete de Python para admitir tanto "
+"la filtración en PyPI como los metadatos que un instalador de paquetes necesita para "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1538]:
```suggestion
"los filtros en PyPI como los metadatos que un instalador de paquetes necesita para "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 3]
```diff
@@ -1510,72 +1520,90 @@ msgid ""
...
-msgstr ""
+msgstr "✨ 4. Publicar en PyPI y Conda ✨"
 
 #: ../../package-structure-code/intro.md:66
 msgid ""
 "If you have a pure Python package, it's a straight forward process to "
 "publish to both PyPI and then a Conda channel such as conda-forge. Learn "
 "more here."
 msgstr ""
+"Si tiene un paquete de Python puro, es un proceso sencillo publicar en PyPI y luego en "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1563]:
```suggestion
"Si tiene un paquete de Python puro, publicar en PyPI y luego en "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 4]
```diff
@@ -1510,72 +1520,90 @@ msgid ""
...
+"un canal de Conda como conda-forge. Aprenda más aquí."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1564]:
```suggestion
"un canal de Conda como conda-forge es un proceso sencillo. Aprenda más aquí."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 5]
```diff
@@ -1587,33 +1615,44 @@ msgid ""
 " and for anyone who is just getting started with creating a Python "
 "package."
 msgstr ""
+"Si está considerando enviar un paquete para revisión por pares, eche un vistazo a las [comprobaciones "
+"mínimos del editor](https://www.pyopensci.org/software-"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1619]:
```suggestion
"mínimas del editor](https://www.pyopensci.org/software-"
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 6]
```diff
@@ -1587,33 +1615,44 @@ msgid ""
...
 msgid "In this section of our Python packaging guide, we:"
-msgstr ""
+msgstr "En esta sección de nuestra guía de empaquetado de Python, nosotros:"
 
 #: ../../package-structure-code/intro.md:113
 msgid ""
 "Provide an overview of the options available to you when packaging your "
 "tool."
 msgstr ""
+"Proporcionamos una descripción general de las opciones disponibles para empaquetar su herramienta."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1638]:
```suggestion
"Proporcionamos una descripción general de las opciones disponibles para empaquetar su código."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 7]
```diff
@@ -1587,33 +1615,44 @@ msgid ""
...
+"Sugerimos herramientas y enfoques que satisfacen sus necesidades y también respaldan los estándares existentes."
 
 #: ../../package-structure-code/intro.md:117
 msgid ""
 "Suggest tools and approaches that will allow you to expand upon a "
 "workflow that may begin as a pure Python tool and evolve into a tool that"
 " requires addition layers of complexity in the packaging build."
 msgstr ""
+"Sugerimos herramientas y enfoques que le permitirán ampliar un flujo de trabajo que puede comenzar "
+"como una herramienta de Python puro y evolucionar hacia una herramienta que requiere capas adicionales "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1654]:
```suggestion
"como código en Python puro y evolucionar hacia un código con capas adicionales "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 8]
```diff
@@ -1651,40 +1701,50 @@ msgid ""
...
+"Sin embargo, algunos paquetes científicos tienen requisitos complejos, ya que pueden necesitar soportar "
+"extensiones o herramientas escritas en otros lenguajes como C o C++."
 
 #: ../../package-structure-code/intro.md:139
 msgid ""
 "To support the many different uses of Python, there are many ways to "
 "create a Python package. In this guide, we suggest packaging approaches "
 "and tools based on:"
 msgstr ""
+"Para apoyar los muchos usos diferentes de Python, hay muchas formas de crear un paquete de Python. "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1714]:
```suggestion
"Para satisfacer todos los usos posibles del lenguaje, hay muchas formas de crear un paquete de Python. "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 9]
```diff
@@ -1651,40 +1701,50 @@ msgid ""
...
 msgid "Tools that we think are well maintained and documented."
-msgstr ""
+msgstr "Heramientas que creemos que están bien mantenidas y documentadas."
 
 #: ../../package-structure-code/intro.md:145
 msgid ""
 "A shared goal of standardizing packaging approaches across this "
 "(scientific) Python ecosystem."
 msgstr ""
+"Un objetivo compartido de estandarizar los enfoques de empaquetado en todo este ecosistema de Python (científico)."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1733]:
```suggestion
"Un objetivo compartido de estandarizar los enfoques de empaquetado en todo el ecosistema de Python científico."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 10]
```diff
@@ -1651,40 +1701,50 @@ msgid ""
...
 
 #: ../../package-structure-code/intro.md:148
 msgid ""
 "Here, we also try to align our suggestions with the most current, "
 "accepted [Python community](https://packaging.python.org/en/latest/) and "
 "[scientific community](https://scientific-python.org/specs/)."
 msgstr ""
+"Aquí, también intentamos alinear nuestras sugerencias con los más actuales "
+"y aceptados [comunidad de Python](https://packaging.python.org/en/latest/) y "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1742]:
```suggestion
"y aceptadas [comunidad de Python](https://packaging.python.org/en/latest/) y "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 11]
```diff
@@ -1651,40 +1701,50 @@ msgid ""
...
 msgstr ""
+"Un objetivo compartido de estandarizar los enfoques de empaquetado en todo este ecosistema de Python (científico)."
 
 #: ../../package-structure-code/intro.md:148
 msgid ""
 "Here, we also try to align our suggestions with the most current, "
 "accepted [Python community](https://packaging.python.org/en/latest/) and "
 "[scientific community](https://scientific-python.org/specs/)."
 msgstr ""
+"Aquí, también intentamos alinear nuestras sugerencias con los más actuales "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1741]:
```suggestion
"Aquí, también intentamos alinear nuestras sugerencias con las prácticas más actuales "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 12]
```diff
@@ -1693,6 +1753,9 @@ msgid ""
 "package to be reviewed and accepted into our pyOpenSci open source "
 "ecosystem."
 msgstr ""
+"Las sugerencias para la distribución de paquetes en esta sección se hacen con la intención de ser útiles; "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1756]:
```suggestion
"Las sugerencias para la distribución de paquetes en esta sección se hacen con la intención de ayudarle; "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 13]
```diff
@@ -1702,6 +1765,11 @@ msgid ""
 "to/author-guide.html#) if you are looking for pyOpenSci's Python package "
 "review requirements!"
 msgstr ""
+"¡Consulte nuestra [página de alcance de paquete](https://www.pyopensci.org"
+"/software-peer-review/about/package-scope.html) y [requisitos de revisión "
+"en nuestra guía para autores](https://www.pyopensci.org/software-peer-review/how-"
+"to/author-guide.html#) si está buscando los requisitos de revisión de paquetes "
+"de Python de pyOpenSci!"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1772]:
```suggestion
"que aplica pyOpenSci!"
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 14]
```diff
@@ -1702,6 +1765,11 @@ msgid ""
 "to/author-guide.html#) if you are looking for pyOpenSci's Python package "
 "review requirements!"
 msgstr ""
+"¡Consulte nuestra [página de alcance de paquete](https://www.pyopensci.org"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1768]:
```suggestion
"¡Consulte nuestra [página con el scope que cubre PyOpenSci](https://www.pyopensci.org"
```

**Review by flpm — CHANGES_REQUESTED** [review]
One more down! Thanks @RobPasMue for helping with this.

**Comment by RobPasMue** [comment]
Thank you once again for reviewing @flpm!

**Review by flpm — APPROVED** [review]
LGTM

**Pull request merged**